resources/views/clients/edit.blade.php

```diff
@@ -0,0 +1,190 @@
+<x-app-layout>
+    <x-slot name="header">
+        <h2 class="font-semibold text-xl text-gray-800 leading-tight">
+            {{ __('Edit client') }}
+        </h2>
+    </x-slot>
+
+
+    <div class="py-12">
+        <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
+            @if($errors->any())
+                <ul class="max-w-7xl mx-auto sm:px-6 lg:px-8 bg-red-700 text-white font-extrabold">
+                    @foreach ($errors->all() as $error)
+                        <li>{{ $error }}</li>
+                    @endforeach
+                </ul>
+            @endif
+            @if(session('message'))
+                <ul class="max-w-7xl mx-auto sm:px-6 lg:px-8 bg-green-700 text-white font-extrabold">
+                    <li>{{ session('message') }}</li>
+                </ul>
+            @endif
+            <form action="{{url('/clients/'.$client->id)}}" method="POST">
+                @csrf
+                @method('PUT')
+                <input name="id" value="{{old('id',$client->id)}}" type="hidden"/>
```

Comment:
@faxunil why you add those hidden fields? The id is already part of the URL, and created_by shouldn't be passed from the form, it should be automatic.